lib/interactive_viewer.ml

```diff
@@ -52,20 +46,47 @@ let current_operation patches =
   | Some p -> ui_of_operation p.Patch.operation
   | None -> W.string "No operation"
 
+let calculate_add_remove lines =
+  List.fold_left
+    (fun (add, rem) line ->
+      match line with
+      | `Their _ -> (add + 1, rem)
+      | `Mine _ -> (add, rem + 1)
+      | _ -> (add, rem))
```

Comment:
It is good practice to try to have exhaustive match (without wildcards) as much as possible. This helps detect when there is a breaking change: for instance, the patch library add a 4th constructor. This 4th constructor would be included in the  `_` joker case, and the compiler would not help us detect that there is a change and that we need to handle it. With an "exhaustive" match, the addition of the 4th constructor would raise a "non-exhaustive match" warning and help us detect the change, so we can handle it!